packages/perseus/src/widgets/expression.tsx

```diff
@@ -537,32 +554,44 @@ export class Expression extends React.Component<Props, ExpressionState> {
         );
     };
 
-    render():
-        | React.ReactNode
-        | React.ReactElement<React.ComponentProps<"div">> {
+    render() {
         if (this.props.apiOptions.customKeypad) {
             return (
-                <KeypadInput
-                    // eslint-disable-next-line react/no-string-refs
-                    ref="input"
-                    value={this.props.value}
-                    keypadElement={this.props.keypadElement}
-                    onChange={this.changeAndTrack}
-                    onFocus={() => {
-                        // this.props.keypadElement should always be set
-                        // when apiOptions.customKeypad is set, but how
-                        // to convince TypeScript of this?
-                        this.props.keypadElement?.configure(
-                            this.props.keypadConfiguration,
-                            () => {
-                                if (this._isMounted) {
-                                    this._handleFocus();
-                                }
-                            },
-                        );
-                    }}
-                    onBlur={this._handleBlur}
-                />
+                <View style={{padding: "15px 4px 0"}}>
```

Comment:
nit: not sure if this is an agreed upon style guidance, I would highly recommend placing style objects as const at the top of the file and injecting it here. I find it makes it easier to find all the style values instead of searching through the React code.